### DISCUSSION TITLE
HmIP-BSM - Taster-Event

### DISCUSSION THREAD

**Discussion by djiwondee** [rdmtc/RedMatic#445] [Q&A]
Hallo,

ist es möglich, mit einer der RedMatic CCU-Nodes (RPC-Event hätte ich vermutet) wirklich jeden Tasten-Druck zu erhalten? Das ist mir bisher nicht gelungen. Ich würde gerne eine Double-Press Message generieren, um in Node-Red noch weiter Funktionen daraus zu triggern.

Danke im Voraus für eine Antwort.

**Comment by Sineos** [comment]
Vielleicht hilft https://github.com/rdmtc/RedMatic/wiki/Faq#tastendruck-erzeugt-keine-events

**Comment by djiwondee** [comment]
@Sineos Danke für den schnellen Hinweis! Mal wieder ein Beispiel für "erst suchen, dann fragen" und "wer lesen kann ist klar im Vorteil".

Habe bisher weder was mit CCU-Programmen noch mit dem Homematic Manager  gemacht. Da werde ich wohl mal einen Blick drauf werfen.

**Comment by djiwondee** [comment]
Funktioniert, wie erwartet:

<img width="1862" alt="image" src="https://user-images.githubusercontent.com/37173958/105870703-d0877b00-5ff8-11eb-8c86-d4ff5bc55c47.png">

<img width="515" alt="image" src="https://user-images.githubusercontent.com/37173958/105870827-f44ac100-5ff8-11eb-8766-321da8e08091.png">

Danke nochmal!!!

**Comment by hobbyquaker** [comment]
Für Tasten-Events muss der Schnittstellenprozess wissen, dass sich jemand dafür interessiert - siehe https://github.com/rdmtc/RedMatic/wiki/Faq#hmip---tastendruck-erzeugt-keine-events. Laut dem letzten Kommentar hat es funktioniert; geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
